livekit-plugins/livekit-plugins-elevenlabs/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@livekit/agents-py-plugin-elevenlabs",
```

Comment:
In the best scenario, this is only used as a placeholder, but nothing is really published to npm

Reply:
yep it's just because package.json's schema requires a name. it will do nothing

Reply:
I still think we should change the name, see my comment above